draft-js-emoji-plugin/src/components/EmojiSelect/Popover/Groups/Group/index.js

```diff
@@ -14,9 +14,32 @@ export default class Group extends Component {
     onEmojiSelect: PropTypes.func.isRequired,
     onEmojiMouseDown: PropTypes.func.isRequired,
     useNativeArt: PropTypes.bool,
+    isActiveGroup: PropTypes.bool.isRequired,
   };
 
-  shouldComponentUpdate = () => false;
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hasRenderedEmoji: props.isActiveGroup,
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.isActiveGroup) {
+      this.setState({
+        hasRenderedEmoji: true,
+      });
+    }
+  }
+
+  shouldComponentUpdate = (nextProps) => {
+    // If the emoji have been rendered once, never update again
+    if (this.state.hasRenderedEmoji) {
```

Comment:
why can't we just check `nextProps.isActiveGroup` here?

Reply:
If I remember correctly, the component was being re-rendered at some points when `isActiveGroup` was `false`. There didn't appear to be any good reasons for these re-renders, so I used the `hasRenderedEmoji` state to prevent the (I think) unnecessary re-renders.